frontend/src/views/Game.tsx

```diff
@@ -38,24 +42,36 @@ function GamePage() {
 
   const firstProblem = problems?.[0];
 
+  const onSecondaryPanelSizeChange = () => {
+    const event = new Event('secondaryPanelSizeChange');
+    window.dispatchEvent(event);
+  };
+
   return (
-    <div>
-      <FlexContainer>
-        <FlexInfoBar>
-          Room:
-          {' '}
-          {room ? room.roomId : 'No room joined'}
-        </FlexInfoBar>
-        <FlexPanel>
-          <ProblemHeaderText>{ firstProblem?.name }</ProblemHeaderText>
-          <Text>{ firstProblem?.description }</Text>
-          { error ? <ErrorMessage message={error} /> : null }
-        </FlexPanel>
-        <FlexPanel>
-          <Editor height="100vh" language="javascript" />
-        </FlexPanel>
-      </FlexContainer>
-    </div>
+    <FlexContainer>
+      <FlexInfoBar>
+        Room:
+        {' '}
+        {room ? room.roomId : 'No room joined'}
+      </FlexInfoBar>
+      <SplitterContainer>
+        <SplitterLayout
+          onSecondaryPaneSizeChange={onSecondaryPanelSizeChange}
+          percentage
+          primaryMinSize={20}
+          secondaryMinSize={35}
+        >
+          <div>
+            <ProblemHeaderText>{firstProblem?.name}</ProblemHeaderText>
+            <Text>{firstProblem?.description}</Text>
+            {error ? <ErrorMessage message={error} /> : null}
+          </div>
+          <div>
+            <Editor height="100vh" language="javascript" />
+          </div>
```

Comment:
Could you add some borders to the `div`s? And any other small changes to start to make the design looks slightly more like [our Site Mockup](https://docs.google.com/presentation/d/1AS2TdFuCMdbHUY4Z_EcZNR3tizKIvc6hV-BG7ZnLqzY/edit#slide=id.g91bce4b29d_1_5)?

Reply:
@helenli8181 ?

Reply:
Yep, what I just pushed fixes this!